extensions/objects/templates/unicode-string-editor.component.spec.ts

```diff
@@ -55,9 +55,12 @@ describe('UnicodeStringEditorComponent', () => {
     spyOn(component, 'closeEditor').and.callThrough();
     spyOn(component.componentSubscriptions, 'add');
     component.active = true;
+    component.alwaysEditable = false;
 
     // Pre-check.
     expect(component.active).toBe(true);
+    // There is no pre-check for alwaysEditable since it's state does not change
```

Comment:
Hi @gp201, I don't understand this comment. If you are not expecting the state to change, why is it being tested?

If you need it as a precondition for subscription.add, that may be OK, but please explain the linkage more clearly. At first glance it is not clear what alwaysEditable has to do with subscription.add.

(Also, nit: it's --> its)

Reply:
Done

Reply:
Same note, still. Why is its being false a precondition for subscription.add? Don't just assert it, try to explain it.

Reply:
Is the modified comment given below ok?
```
// subscription.add is called only if the text input is not always editable.
// The alwaysEditable variable is tested since it is a pre-condition
// for subscription.add. 
```

Reply:
It's closer, but establish the link:

```
// subscription.add is called only if the text input is not always editable.
// Therefore, we verify that the value of alwaysEditable is false before checking
// subscription.add. 
```

Reply:
Done. Thank you for your guidance!